diff --git a/node_modules/babel-plugin-react-native-web/src/moduleMap.js b/node_modules/babel-plugin-react-native-web/src/moduleMap.js
index a83dd77..a538da6 100644
--- a/node_modules/babel-plugin-react-native-web/src/moduleMap.js
+++ b/node_modules/babel-plugin-react-native-web/src/moduleMap.js
@@ -55,7 +55,7 @@ module.exports = {
   VirtualizedList: true,
   YellowBox: true,
   createElement: true,
-  findNodeHandle: true,
+  // findNodeHandle: true,
   processColor: true,
   render: true,
   unmountComponentAtNode: true,
