diff --git a/node_modules/expo-image/build/ExpoImage.web.d.ts b/node_modules/expo-image/build/ExpoImage.web.d.ts
index a9dc846..50839d9 100644
--- a/node_modules/expo-image/build/ExpoImage.web.d.ts
+++ b/node_modules/expo-image/build/ExpoImage.web.d.ts
@@ -1,3 +1,5 @@
 import { ImageNativeProps } from './Image.types';
+declare const ExpoImageModule: any;
+export { ExpoImageModule };
 export default function ExpoImage({ source, placeholder, contentFit, contentPosition, placeholderContentFit, onLoad, transition, onError, responsivePolicy, onLoadEnd, priority, blurRadius, recyclingKey, ...props }: ImageNativeProps): JSX.Element;
 //# sourceMappingURL=ExpoImage.web.d.ts.map
\ No newline at end of file
diff --git a/node_modules/expo-image/build/ExpoImage.web.js b/node_modules/expo-image/build/ExpoImage.web.js
index 6a5a231..1dccf68 100644
--- a/node_modules/expo-image/build/ExpoImage.web.js
+++ b/node_modules/expo-image/build/ExpoImage.web.js
@@ -18,6 +18,7 @@ function onLoadAdapter(onLoad) {
         });
     };
 }
+export const ExpoImageModule = null;
 function onErrorAdapter(onError) {
     return ({ source }) => {
         onError?.({
diff --git a/node_modules/expo-image/build/Image.js b/node_modules/expo-image/build/Image.js
index c444806..eb0558e 100644
--- a/node_modules/expo-image/build/Image.js
+++ b/node_modules/expo-image/build/Image.js
@@ -4,6 +4,7 @@ import ExpoImage, { ExpoImageModule } from './ExpoImage';
 import { resolveContentFit, resolveContentPosition, resolveTransition } from './utils';
 import { resolveSources } from './utils/resolveSources';
 let loggedDefaultSourceDeprecationWarning = false;
+export { ExpoImage, ExpoImageModule };
 export class Image extends React.PureComponent {
     /**
      * Preloads images at the given urls that can be later used in the image view.
