diff --git a/node_modules/expo-router/src/aasa.ts b/node_modules/expo-router/src/aasa.ts
index 4070e1a..91e4af5 100644
--- a/node_modules/expo-router/src/aasa.ts
+++ b/node_modules/expo-router/src/aasa.ts
@@ -17,7 +17,7 @@ function getWebUrlsFromManifest() {
       );
     });
 
-  return [...new Set(applinks)];
+  return Array.from(new Set(applinks));
 }
 
 export function getAllWebRedirects(
diff --git a/node_modules/expo-router/src/link/linking.ts b/node_modules/expo-router/src/link/linking.ts
index 392c848..37e0e47 100644
--- a/node_modules/expo-router/src/link/linking.ts
+++ b/node_modules/expo-router/src/link/linking.ts
@@ -81,6 +81,7 @@ export function addEventListener(listener: (url: string) => void) {
     | undefined;
 
   // Storing this in a local variable stops Jest from complaining about import after teardown
+  // @ts-ignore
   const removeEventListener = Linking.removeEventListener?.bind(Linking);
 
   return () => {
