diff --git a/node_modules/expo-image/build/ExpoImage.web.d.ts b/node_modules/expo-image/build/ExpoImage.web.d.ts
index b7feabf..3139fa6 100644
--- a/node_modules/expo-image/build/ExpoImage.web.d.ts
+++ b/node_modules/expo-image/build/ExpoImage.web.d.ts
@@ -1,3 +1,5 @@
 import { ImageNativeProps } from './Image.types';
+declare const ExpoImageModule: any;
+export { ExpoImageModule };
 export default function ExpoImage({ source, placeholder, contentFit, contentPosition, placeholderContentFit, onLoad, transition, onError, responsivePolicy, onLoadEnd, priority, blurRadius, ...props }: ImageNativeProps): JSX.Element;
 //# sourceMappingURL=ExpoImage.web.d.ts.map
\ No newline at end of file
diff --git a/node_modules/expo-image/build/ExpoImage.web.js b/node_modules/expo-image/build/ExpoImage.web.js
index 79ee8ed..10f9e88 100644
--- a/node_modules/expo-image/build/ExpoImage.web.js
+++ b/node_modules/expo-image/build/ExpoImage.web.js
@@ -29,6 +29,7 @@ const setCssVariables = (element, size) => {
     element?.style.setProperty('--expo-image-width', `${size.width}px`);
     element?.style.setProperty('--expo-image-height', `${size.height}px`);
 };
+export const ExpoImageModule = null;
 export default function ExpoImage({ source, placeholder, contentFit, contentPosition, placeholderContentFit, onLoad, transition, onError, responsivePolicy, onLoadEnd, priority, blurRadius, ...props }) {
     const { aspectRatio, backgroundColor, transform, borderColor, ...style } = props.style ?? {};
     const imagePlaceholderContentFit = placeholderContentFit || 'scale-down';
diff --git a/node_modules/expo-image/build/Image.js b/node_modules/expo-image/build/Image.js
index c444806..eb0558e 100644
--- a/node_modules/expo-image/build/Image.js
+++ b/node_modules/expo-image/build/Image.js
@@ -4,6 +4,7 @@ import ExpoImage, { ExpoImageModule } from './ExpoImage';
 import { resolveContentFit, resolveContentPosition, resolveTransition } from './utils';
 import { resolveSources } from './utils/resolveSources';
 let loggedDefaultSourceDeprecationWarning = false;
+export { ExpoImage, ExpoImageModule };
 export class Image extends React.PureComponent {
     /**
      * Preloads images at the given urls that can be later used in the image view.
diff --git a/node_modules/expo-image/build/web/style.js b/node_modules/expo-image/build/web/style.js
index 7fa600c..eeb9cf0 100644
--- a/node_modules/expo-image/build/web/style.js
+++ b/node_modules/expo-image/build/web/style.js
@@ -127,8 +127,10 @@ const style = `
 }
 `;
 export default function loadStyle() {
+  if (typeof document !== 'undefined') {
     const styleTag = document.createElement('style');
     styleTag.innerHTML = style;
     document.head.appendChild(styleTag);
+  }
 }
 //# sourceMappingURL=style.js.map
\ No newline at end of file
