diff --git a/node_modules/metro-file-map/src/watchers/NodeWatcher.js b/node_modules/metro-file-map/src/watchers/NodeWatcher.js
index a9403c6..957a23d 100644
--- a/node_modules/metro-file-map/src/watchers/NodeWatcher.js
+++ b/node_modules/metro-file-map/src/watchers/NodeWatcher.js
@@ -86,15 +86,17 @@ module.exports = class NodeWatcher extends EventEmitter {
     if (this.watched[dir]) {
       return false;
     }
-    const watcher = fs.watch(
+    if (!/.git|node_modules/.test(dir)) {
+      const watcher = fs.watch(
       dir,
       {
         persistent: true,
       },
       (event, filename) => this._normalizeChange(dir, event, filename)
-    );
-    this.watched[dir] = watcher;
-    watcher.on("error", this._checkedEmitError);
+      );
+      this.watched[dir] = watcher;
+      watcher.on("error", this._checkedEmitError);
+    }
     if (this.root !== dir) {
       this._register(dir, "d");
     }
