diff --git a/node_modules/next/dist/client/image.d.ts b/node_modules/next/dist/client/image.d.ts
index 2a47d83..74e6b95 100644
--- a/node_modules/next/dist/client/image.d.ts
+++ b/node_modules/next/dist/client/image.d.ts
@@ -14,7 +14,7 @@ export interface StaticImageData {
     blurWidth?: number;
     blurHeight?: number;
 }
-interface StaticRequire {
+export interface StaticRequire {
     default: StaticImageData;
 }
 type StaticImport = StaticRequire | StaticImageData;
