diff --git a/node_modules/expo-font/build/FontLoader.web.js b/node_modules/expo-font/build/FontLoader.web.js
index d287341..56d5bf8 100644
--- a/node_modules/expo-font/build/FontLoader.web.js
+++ b/node_modules/expo-font/build/FontLoader.web.js
@@ -6,7 +6,7 @@ function uriFromFontSource(asset) {
         return asset || null;
     }
     else if (typeof asset === 'object') {
-        return asset.uri || asset.localUri || null;
+        return asset.uri || asset.localUri || asset.default || null;
     }
     return null;
 }
diff --git a/node_modules/expo-font/src/Font.types.ts b/node_modules/expo-font/src/Font.types.ts
index 2ba505c..efa339f 100644
--- a/node_modules/expo-font/src/Font.types.ts
+++ b/node_modules/expo-font/src/Font.types.ts
@@ -13,12 +13,13 @@ export type FontSource = string | number | Asset | FontResource;
  * with [`loadAsync`](#loadasync).
  */
 export type FontResource = {
-  uri: string | number;
+  uri?: string | number;
   /**
    * _Web Only._ Sets the [`font-display`](#fontdisplay) property for a given typeface in the
    * browser.
    */
   display?: FontDisplay;
+  default?: string;
 };
 
 // @needsAudit
diff --git a/node_modules/expo-font/src/FontLoader.web.ts b/node_modules/expo-font/src/FontLoader.web.ts
index 0e7955e..e3abe75 100644
--- a/node_modules/expo-font/src/FontLoader.web.ts
+++ b/node_modules/expo-font/src/FontLoader.web.ts
@@ -9,7 +9,7 @@ function uriFromFontSource(asset: any): string | null {
   if (typeof asset === 'string') {
     return asset || null;
   } else if (typeof asset === 'object') {
-    return asset.uri || asset.localUri || null;
+    return asset.uri || asset.localUri || asset.default || null;
   }
   return null;
 }
